docs/cli.rst

```diff
@@ -204,6 +204,17 @@ Options
 
     Query orbit products form the GNSS end-point ("https://scihub.copernicus.eu/gnss").
 
+.. option:: --fmt <format string>
+
+    Specify a custom format to print results. The format string shall
+    be compatible with the Python "Format Specification Mini-Language".
+
+    A complete list keys that can be used for a specific sentinel mission
+    can be e.g. obtained looking at the "properties" of the output produced by
+    the ``--footprints`` flag  - (maybe with ``--limit 1``).
```

Comment:
Should include the dash in `--footprints -`, which directs the output to stdout. I would still also include a subjective list of common and likely useful keywords as well. Suggested wording:
```suggestion
    Some common keywords for substitution are:
      'uuid', 'title', 'summary', 'link', 'size', 'platformname', 'producttype',
      'beginposition', 'instrumentshortname', 'cloudcoverpercentage',
      'orbitdirection', 'relativeorbitnumber', 'footprint'.

    For a complete set of available keywords see the "properties" output from a relevant query with ``--footprints -`` (and possibly ``--limit 1``) appended.
```

Reply:
@valgur your suggested wording is OK for me, just two questions:

* what about the ``identifier`` and/or ``filename`` keys? These are the primary reason why I needed to add this feature. Do you think that they can be added to the list?
* you suggested a rewording for the sphinx documentation. The CLI help message and the README shall be updated accordingly?

@j08lue the above changes are OK for you? Do you have any comment?

Reply:
`identifier` is identical to `title`. I suppose `identifier` is a bit clearer, so you can replace it with that. Feel free to add `filename` as well.
I would add the more detailed info to the CLI help message, too, sure, but not to the README.

Reply:
ok, thanks